pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -208,6 +208,10 @@ func (r *Builder) PodEnvironment(vmRef ref.Ref, sourceSecret *core.Secret) (env
 			Name:  "V2V_fingerprint",
 			Value: fingerprint,
 		},
+		core.EnvVar{
+			Name:  "V2V_extra_args",
+			Value: settings.Settings.Migration.Virt,
+		},
```

Comment:
you can take https://github.com/kubev2v/forklift/pull/852/files#diff-c14563539716a81d759e3204377a4a3f07328ee75d3fb4d9bb5bdc74f7791538R218 as a reference, i think this is why it fails.

Reply:
yeah, that was problematic as `settings.Settings.Migration.Virt` doesn't exist - changed